src/test/xmeta/CMakeLists.txt

```diff
@@ -5,17 +5,32 @@ project(test_xmeta)
 # using /MD flag for antlr4_runtime (for Visual C++ compilers only)
 set(ANTLR4_WITH_STATIC_CRT ON)
 
+include_directories(
+    ${CMAKE_SOURCE_DIR}/tool/xmeta/
+)
+
 add_executable(test_xmeta "")
 
+target_link_libraries(test_xmeta 
+    WindowsApp.lib
+    comsuppw.lib
+)
+
+
 if (EXISTS "${CMAKE_SOURCE_DIR}/test/xmeta/example/midl3test")
```

Comment:
Is this stuff still being used?

Reply:
Yup. The test idl files are not really public so they are not in the repo right now.